test/Sentry.Maui.Tests/SentryMauiOptionsTests.cs

```diff
@@ -52,12 +52,32 @@ public void CacheDirectoryPath_Default()
 #endif
     }
 
+#if ANDROID
     [Fact]
-    public void AttachScreenshots_Default()
+    public void HandlerStrategy_Default()
     {
+        // Arrange
+        var expected = Android.LogCatIntegrationType.None;
+        var options = new SentryMauiOptions();
+
+        // Assert
+        Assert.Equal(expected, options.Android.LogCatIntegration);
```

Comment:
Why do we have logcat tests on the NDK PR?

Reply:
As mentioned in the description, this PR is mostly to fix #3461. The issue was that the logcat file was not attached.

So, it's crucial to test that the logcat file is being attached properly

Reply:
@bricefriha I think [that comment](https://github.com/dotnet/android/issues/9055#issuecomment-2261347912) describes a specific situation in which logcat logs aren't sent. More generally, as long as we're not dealing with SEGSEGV exceptions like that, we should be getting logcat reports come through though, so it should be possible to test this fix without mixing the two PRs right?